WordPressKit/WordPressKit/TaxonomyServiceRemoteXMLRPC.m

```diff
@@ -102,6 +104,39 @@ - (void)createTag:(RemotePostTag *)tag
                          } failure:failure];
 }
 
+- (void)updateTag:(RemotePostTag *)tag
+          success:(nullable void (^)(RemotePostTag *tag))success
+          failure:(nullable void (^)(NSError *error))failure
+{
+    NSMutableDictionary *extraParameters = [NSMutableDictionary dictionary];
+    [extraParameters setObject:tag.tagID ?: [NSNull null] forKey:TaxonomyXMLRPCIDParameter];
+    [extraParameters setObject:tag.name ?: [NSNull null] forKey:TaxonomyXMLRPCNameParameter];
+    [extraParameters setObject:tag.description ?: [NSNull null] forKey:TaxonomyXMLRPCDescriptionParameter];
+    
+    [self editTaxonomyWithType:TaxonomyXMLRPCTagIdentifier
+                    parameters:extraParameters success:^(BOOL response) {
+                        if (success) {
+                            success(tag);
+                        }
+                    } failure:failure];
+}
+
+- (void)deleteTag:(RemotePostTag *)tag
+          success:(nullable void (^)(void))success
+          failure:(nullable void (^)(NSError *error))failure
+{
+    NSMutableDictionary *extraParameters = [NSMutableDictionary dictionary];
+    [extraParameters setObject:tag.tagID ?: [NSNull null] forKey:TaxonomyXMLRPCIDParameter];
```

Comment:
Same issue here as in `editTag:success:failure:` with where the term_id is passed.